src/js/components/locations-configuration/ZoneAndBinLocations.jsx

```diff
@@ -157,7 +263,34 @@ class ZoneAndBinLocations extends Component {
                   zoneData: this.state.zoneData.filter(location => location.id !== locationId),
                 });
               })
-              .catch(() => Promise.reject(new Error(this.props.translate('react.locationsConfiguration.editZone.error.label', 'Could not edit zone location'))));
+              .catch(() => Promise.reject(new Error(this.props.translate('react.locationsConfiguration.deleteZone.error.label', 'Could not delete zone location'))));
+          },
+        },
+        {
+          label: this.props.translate('react.default.no.label', 'No'),
+        },
+      ],
+    });
+  }
+
+  deleteBinLocation(locationId) {
```

Comment:
You can combine the `deleteZoneLocation` and `deleteBinLocation` into one `deleteLocation`